src/MfaVerifyController.js

```diff
@@ -106,10 +109,12 @@ function (Okta, BaseLoginController, TOTPForm, YubikeyForm, SecurityQuestionForm
             }
           });
         }
-        // Set the rememberDevice on the TOTP factor since it is stored as backupFactor.
-        this.listenTo(this.model, 'change:rememberDevice', function (model, rememberDevice) {
-          model.get('backupFactor').set('rememberDevice', rememberDevice);
-        });
+        // Set rememberDevice on the backup factor (totp) if available
```

Comment:
yes, that is set in the model that is handling push factor. We listen to the changes in that model, and then set the same value for totp as well (backup factor). This is because the user sees Okta Verify as one factor